pkg/api/unversioned/deep_copy_generated.go

```diff
@@ -21,37 +21,28 @@ limitations under the License.
 package unversioned
 
 import (
-	time "time"
-
 	conversion "k8s.io/kubernetes/pkg/conversion"
+	time "time"
```

Comment:
It was that revelation that kicked off this whole PR.  I'm going to close this comment for cleanliness.